app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -42,6 +42,7 @@ def setup_form_object(step)
           begin
             form_name.constantize
           rescue NameError => x
```

Comment:
We need holding pages that render when one gets to a step that hasn't been built yet. Otherwise we'd have to build the steps in sequence and we'd not be able to test completion of a page (as success would lead to rendering of the next page which hadn't been built).

Without the generic locale file, the holding pages fail due to them not having heading locales defined.

We can get rid of the generic locale file when we no longer need holding pages.